src/plugins/kibana_utils/docs/store/redux.md

```diff
@@ -0,0 +1,19 @@
+# Redux
+
+Internally `createStore()` uses Redux to manage the state. When you call `store.get()`
+it is actually calling the Redux `.getState()` method. When you execute a mutation
+it is actually dispatching a Redux action.
+
+You can access Redux *store* using `.redux`.
+
+```ts
+store.redux;
+```
+
+But you should never need it, if you think you do, consult with [`@elastic/kibana-app-arch`](https://github.com/orgs/elastic/teams/kibana-app-arch).
```

Comment:
Is this link accessible to non-Elastic members? If not then maybe we should remove it, for the same reason we avoid linking to private repos.

Reply:
Thanks, I did not realize that was a private link, removed it.